force-app/main/default/triggers/CQ_GJ_Child_Object_Trigger.trigger

```diff
@@ -0,0 +1,10 @@
+trigger CQ_GJ_Child_Object_Trigger on CQ_GJ_SQX_Child_Object__c (before update, before delete, before insert) {
```

Comment:
no condition for before insert ?

Reply:
removed that i have written that by mistake